.github/ISSUE_TEMPLATE/mandatory-securit-scans.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Mandatory Securit Scans
+about: Scan newly created repository.
+title: ''
+labels: area/security
+assignees: ''
+
+---
+
+Please follow detailed process description: https://wiki.one.int.sap/wiki/display/kyma/Add+new+repositories+to+the+mandatory+scanners
+
+Tasks:
+-[ ] Add Whitesource Prow Job. [requestor]
+-[ ] Protecode: provide a list of images to scan. [requestor]
```

Comment:
This sounds like requestor should provide a list in this issue, while the requirement is to provide a list of images every time they need to be scanned or changed. It's rather about integrating a source with Jenkins pipeline.